Simplified.xcodeproj/project.pbxproj

```diff
@@ -6030,6 +6044,10 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				LIBRARY_SEARCH_PATHS = (
+					"$(inherited)",
+					"$(PROJECT_DIR)/Carthage/Build/iOS/ZIPFoundation.framework",
```

Comment:
this should not be necessary, because the all other frameworks are in the same directory. Same for the one below.

Reply:
Removed, not sure why Xcode added that